tools/lint/src/main/java/com/grab/lint/LintAnalyzeCommand.kt

```diff
@@ -13,13 +21,21 @@ class LintAnalyzeCommand : LintBaseCommand() {
         workingDir: Path,
         projectXml: File,
         tmpBaseline: File,
-        lintBaselineHandler: LintBaselineHandler
     ) {
         val cliArgs = (defaultLintOptions + listOf(
             "--cache-dir", workingDir.resolve("cache").pathString,
             "--project", projectXml.toString(),
             "--analyze-only" // Only do analyze
         )).toTypedArray()
         LintCli().run(cliArgs)
+        sanitizePartialResults()
+    }
+
+    private fun sanitizePartialResults() = runBlocking {
+        partialResults.listFiles()?.asFlow()?.flatMapMerge { partialResult ->
```

Comment:
Can be written as 

```kotlin
partialResults.walkTopDown().filter { it.isFile }.asFlow()
```

and you have avoid `?` as well this handles sub folder in case lint implementation changes later.